pods/account/Dockerfile

```diff
@@ -6,6 +6,7 @@ RUN npm install --ignore-scripts=false --verbose bufferutil utf-8-validate @mong
 
 RUN apt-get update
 RUN apt-get install libjemalloc2
+RUN apt-get clean
```

Comment:
Official Debian and Ubuntu images automatically run apt-get clean, so explicit invocation is not required.
https://docs.docker.com/build/building/best-practices/

Reply:
Nice 😉